README.md

```diff
@@ -50,49 +50,64 @@ Sometimes it can be useful to override these options without changing or recompi
 
 Using Maven:
 
-    mvn -Dcucumber.options="..." test
+    mvn -Dcucumber.features="..." -Dcucumber.glue="..." test
 
 Using Gradle:
 
-    gradlew -Dcucumber.options="..." test
-
-Let's look at some things you can do with `cucumber.options`. Try this:
-
-    -Dcucumber.options="--help"
-
-That should list all the available options.
```

Comment:
It's still possible to list all available options using the CLI:

```
mvn exec:java                                   \ 
  -Dexec.classpathScope=test                    \
  -Dexec.mainClass=io.cucumber.core.cli.Main    \
  -Dexec.args="--help"
```

Reply:
I stuck at this too, my command doesnt work:

```
mvn test \
  -Dsuite=cucumber \
  -Dplatform=android \
  -Denvironment=sandboxqc \
  -Ddevice-name=emulator \
  -Dcucumber.options="--help"
``

or am I doing it wrong?

Reply:
Your CLI command works only if I have `exec` plugin defined in my POM, do we have a more general way to print it out?

Reply:
You might have to use the full name of the plugin then. `org.codehaus.mojo:exec-maven-plugin:3.0.0:java`

Reply:
Hey it was my mistake, your command run well without modifying POM file

Reply:
I've just updated the README, please take a look :D